README.md

```diff
@@ -402,13 +409,10 @@ The list of fields are:
 
 ### Logging & tracing middleware
 
-`go-sdk` ships with a `Logger` middleware. When used, it assigns a random
-UUID as a `RequestID`, if none exists, and it opens a new span using Datadog's
-`ddtrace` library. The span has a corresponding `SpanID` that is used to easily
-create sub-spans for application performance monitoring (APM). The `RequestID`
-is used for request-scoped logs correlation.
+`go-sdk` ships with a `Logger` middleware. When used, it tries to retrieve the `RequestID`, `TraceID` and `SpanID`
+from the incoming context. Then, middleware assigns those values to the log entries for further correlation.
 
-Example usage of the middleware:
+#### HTTP server middleware:
```

Comment:
nitpick: ✂️ `:`. We don't use it anywhere in headers

Reply:
@terranisu Good catch 👍  Removed ✂️